src/jit/lsra.cpp

```diff
@@ -1868,6 +1868,14 @@ void LinearScan::identifyCandidates()
             continue;
         }
 
+#if FEATURE_MULTIREG_ARGS_OR_RET        
+        if (varDsc->lvIsMultiRegArgOrRet)
+        {
+            varDsc->lvLRACandidate = 0;
+            continue;
+        }        
+#endif
+
```

Comment:
As per my understanding this issue is exposed by the below stmnt
Vector3 ret = nativeCall_PInvoke_Vector3Ret();

Where-in while jit compiling IL stub of PInvoke, we are hitting this assert.

Since the native call is returning Vector3, we must be forcing IR to be of the form var=call.  Is the bug is that var is getting marked only as multiRegArgOrRet but not DoNotEnregister on Unix?

Just for my understanding why would that be a bigger change?  We already to do that in importer.  Looks like in some code path we are missing to set DoNotEnregister? is that the core issue?